README.rst

```diff
@@ -4,6 +4,9 @@ C/C++ Linting Package
 .. |latest-version| image:: https://img.shields.io/github/v/release/cpp-linter/cpp-linter
     :alt: Latest Version
     :target: https://github.com/cpp-linter/cpp-linter/releases
+.. |python-version| image:: https://img.shields.io/pypi/pyversions/cpp-linter
+    :alt: Python Version
+    :target: https://pypi.org/project/cpp-linter
```

Comment:
Was this badge the intended result?
![image](https://github.com/user-attachments/assets/a96e1746-db27-4154-9aa0-5bd90dbf0201)

I thought you were going for a "min python: v3.9"
```suggestion
.. |python-version| image:: https://img.shields.io/badge/Python-v3.9%2B-blue
    :alt: Minimum Python Version
    :target: https://pypi.org/project/cpp-linter
```

which renders: ![image](https://github.com/user-attachments/assets/b7668f38-ffa3-437d-8d9e-da4d3ac2f21e)